.travis.yml

```diff
@@ -0,0 +1,16 @@
+addons:
+  apt:
+    packages:
+      - xvfb
+
+before_install:
+  ## Install latest version of node
+  - nvm install node
+
+install:
+  - export DISPLAY=':99.0'
+  - Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+script:
+  - npm install -g eslint eslint-config-google 
```

Comment:
We can actually remove the entire `script` node here and update `packages.json` to use the `scripts` object. Look at what we did in the main repo